doc/source/guide/plotting.rst

```diff
@@ -127,65 +127,126 @@ plot window blocks the command line terminal and must be closed before doing any
 4.2 plot()
 ----------
 
-For more advanced plotting the base SunPy objects also provide a `~sunpy.map.mapbase.GenericMap.plot` command.
-This command is similar to the pyplot `~matplotlib.pyplot.plot` command in that
-it will create a figure and axes object for you if you haven't already. It
-returns a figure object and does not create a plot window. With the `~matplotlib.figure.Figure` object
-in your hands you can reach in and grab the axes and therefore manipulate the plot.
-Here is a simple example which outputs the same plot as we saw before but with contours added:
+For more advanced plotting the base SunPy objects also provide a
+`~sunpy.map.mapbase.GenericMap.plot` command. This command is similar to the
+pyplot `~matplotlib.pyplot.imshow` command in that it will create a figure and
+axes object for you if you haven't already.
+
+When you create a plot with `~sunpy.map.GenericMap.peek` or
+`~sunpy.map.GenericMap.plot`, if possible SunPy will use `wcsaxes
+<http://wcsaxes.readthedocs.io/>`_ to represent coordinates on the image
+accurately, for more information see :ref:`wcsaxes-plotting`.
+
+Using `~sunpy.map.GenericMap.plot` it is possible to customise the look of the
+plot by combining SunPy and matplotlib commands, for example you can over plot
+contours on the Map:
 
 .. plot::
     :include-source:
 
+    import matplotlib.pyplot as plt
+    import astropy.units as u
+
     import sunpy.map
     import sunpy.data.sample
-    import matplotlib.pyplot as plt
-    from astropy.units import Quantity
-    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    smap.plot()
-    smap.draw_limb()
+
+    aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    aia_map.plot()
+    aia_map.draw_limb()
+
     # let's add contours as well
-    smap.draw_contours(Quantity([10,20,30,40,50,60,70,80,90], 'percent'))
+    aia_map.draw_contours([10,20,30,40,50,60,70,80,90] * u.percent)
```

Comment:
I think for the sake of examples this is clearer, because it can be any numbers you like, we just chose reguarly spaced numbers from 0 to 100!